__tests__/hooks/useFeed.test.js

```diff
@@ -162,7 +162,7 @@ describe('Logic feed should run correctly', () => {
   });
   it('getHeightFooter should run correctly', async () => {
     const {result} = renderHook(useFeed);
-    expect(result.current.getHeightFooter()).toEqual(normalizeFontSizeByWidth(52));
+    expect(result.current.getHeightFooter()).toEqual(normalizeFontSizeByWidth(49));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the expected value of `getHeightFooter` from `normalizeFontSizeByWidth(52)` to `normalizeFontSizeByWidth(49)` might be due to a design change. However, it's important to ensure that this new value is consistent with the actual footer height in the UI. If there are any discrepancies, it could lead to layout issues.


<!-- This is an auto-generated comment by OSS CodeRabbit -->